package-lock.json

```diff
@@ -0,0 +1,261 @@
+{
```

Comment:
sorry, I'm wrong with `npm install`.

The best way to achieve fast and reproducible builds in the CI context is [`npm ci`](https://docs.npmjs.com/cli/ci). However, it requires `package.json` file to exist. It will never write to package.json or any of the package-locks: installs are essentially frozen.

package.json example:
```
{
  "name": "docs",
  "version": "1.0.0",
  "devDependencies": {
      "markdownlint-cli": "0.17.0"
  }
}
```